src/components/AboutCard.jsx

```diff
@@ -21,26 +18,24 @@ function AboutCard(props) {
   const { img, name, description } = props;
 
   return (
-    <StylesProvider injectFirst>
```

Comment:
I removed this as I don't believe it's necessary. It should probably actually be injected last (the default) since other stylesheets could potentially override certain settings